src/index.ts

```diff
@@ -2444,7 +2447,9 @@ function FlatpickrInstance(
       }
     } else {
       self.config[option] = value;
-
+      if (option === "showTodayButton") {
```

Comment:
It seems strange to me that we have this specific if condition here. Can we make the "same" logic as for other options? This `function set(..)` basically defines `self.config`, which is later used by the flatpickr. So based on that logic, we need to handle `buildExtraButtons()`  somewhere else.

You can follow some other settings for reference. Like `weekNumbers` and then adjust the code to call `buildExtraButtons();`.

Reply:
Ps. Also, rename it to `todayButton`. To follow the same naming convention.